woocommerce/woocommerce-admin
# Replace welcome image on marketing tab with svg.

Fixes #4057

Replaced the welcome graphic with a scalable svg.

### Screenshots
Before:
![before](https://cldup.com/STnCkOJAa9.png)

After:
![after](https://cldup.com/7L73-2-2SN-3000x3000.png)

### Detailed test instructions:

- View the Marketing tab
- Ensure the welcome image still displays as expected
